api/views.py

```diff
@@ -11,16 +11,18 @@
 from rest_framework.response import Response
 from rest_framework.renderers import JSONRenderer
 from rest_framework.decorators import action
+from api.serializers import AuthorSerializer, CommentSerializer, FollowersSerializer, PostSerializer, LikesSerializer, RemoteLikeSerializer, RequestSerializer, RemoteRequestSerializer
 from rest_framework.exceptions import PermissionDenied
 from api.serializers import AuthorSerializer, CommentSerializer, FollowersSerializer, PostSerializer, LikesSerializer, RemoteLikeSerializer
```

Comment:
Duplicate using line:
```suggestion
from api.serializers import AuthorSerializer, CommentSerializer, FollowersSerializer, PostSerializer, LikesSerializer, RemoteLikeSerializer, RequestSerializer, RemoteRequestSerializer
from rest_framework.exceptions import PermissionDenied
```